roboball2d/ball/#ball_config.py#

```diff
@@ -0,0 +1,54 @@
+import numpy as np
```

Comment:
Looks like this file is an older version of ball_config.py in the same directory. Can it be removed?